### ISSUE TITLE
Plex MPV Desktop?

### ISSUE THREAD

**Issue by arlinsandbulte** [iwalton3/plex-mpv-shim#25]
Is it possible to have a Plex MPV Desktop app like Jellyfin MPV Desktop?  (thus 'Desktop' window is on one screen for the UI, and the video is played on another screen).
I REALLY like Jellyfin and Jellyfin MPV Desktop, but overall Jellyfin just is not quite fully developed yet, IMO.

I like Jellyfin MPV Desktop for dual display setups where the Jellyfin Desktop window is launched on one display(0), but MPV is configured to play the video fullscreen on another display(1).
This happens AUTOMATICALLY when Jellyfin Desktop is launched, so the user does not need to first choose the cast the display.

So, in a nutshell, I want to be able to open Plex on display(0) to browse & interact & control as normal, with videos being played on display(1) fullscreen.  AND, I don't want to need to first click the 'cast' button to choose to play the video on display(1).  I just want to open 'Plex Desktop' and play a video on my secondary display.

I hope that makes sense.  
Thanks for all the great work on Plex & Jellyfin & MPV!

**Comment by dmgolembiowski** [comment]
Svelte-GTK for Plex MPV Shim!!!!

**Comment by dmgolembiowski** [comment]
Please, @iwalton3 , pretty please

**Comment by iwalton3** [comment]
This is not practical for me to support due to licensing restrictions and the compiled nature of the Plex web app. I plan on making a native UI for Jellyfin MPV Shim in the future. I do not plan on doing the same for Plex unless someone wants to contribute the changes.

**Comment by arlinsandbulte** [comment]
Fair enough.  I understand.
Just one more naïve question: Might it be possible to just wrap the Plex Web interface in a desktop window where the cast setting is automatically set to cast to Plex MPV Shim?
Maybe, if I try I could even do something like that with a AutoHotKey script.  The script could run Plex-MPV-Shim, then open the a browser with Plex web UI, then enable casting....
Thanks again!

**Comment by iwalton3** [comment]
I would say that is possible and you can start with the local connection script which will also make the connection more stable.
